packages/editor/src/components/block-list/block.js

```diff
@@ -93,6 +93,12 @@ export class BlockListBlock extends Component {
 		if ( this.props.isSelected && ! prevProps.isSelected ) {
 			this.focusTabbable( true );
 		}
+
+		// When triggering a multi-selection,
+		// move the focus to the wrapper of the first selected block.
```

Comment:
These comments explain what we're doing (which is pretty plainly obvious in the implementation anyways) but not why. I don't know why we're doing this.

Reply:
We move the focus to ensure that it is not possible to continue editing the initially selected block. This happens for all active elements that allow editing like `RichText`, `PlainText` and all form fields. This behavior ensures that you no longer can modify the content of the selected blocks.